### ISSUE TITLE
Allow adding data when burning inscriptions

### ISSUE THREAD

**Issue by casey** [ordinals/ord#1334]
Depends on #1333.

- Allow adding data to be included in the OP_RETURN
- Display that data on an inscription page if it's been burned. Possibly in the address field, or in an alternative field.

**Comment by haribosan** [comment]
I like it. Either that or a standard "burned" / "OP_RETURN" label if allowing arbitrary data becomes too cumbersome to implement.

**Comment by casey** [comment]
Merged into #1333.
